h/accounts/schemas.py

```diff
@@ -193,13 +199,15 @@ def validator(self, node, value):
 
 class PasswordChangeSchema(CSRFSchema):
     password = password_node(title=_("Current password"), inactive_label=_("Password"))
-    new_password = password_node(title=_("New password"), hide_until_form_active=True)
+    new_password = new_password_node(
```

Comment:
Attention I changed `password_node` for `new_password_node`. The only difference is that the `new_password_node` adds a length validator.